modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -1,44 +1,40 @@
-= Guide to Named Examples in RAML 1.0
+= Guide to Defining Examples in RAML 1.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-If you want to use one or more named examples for a type, trait, or resource type in your RAML 1.0 API specification, you must follow these three rules:
+When defining examples in external files for a type, trait, or resource type in your RAML 1.0 API specification, you should follow these rules:
 
-* To use a single, named example, you must use the `examples` facet when you include the corresponding NamedExample fragment.
+* For a single example, you should use the `example` facet and include a `.yaml` file.
+* For multiple examples, you should always use the `examples` facet and either:
+** define multiple examples, each pointing to individual `.yaml` files, with a single example per file; or
+** include a `.raml` file containing two or more examples defined in a single NamedExample fragment.
 
-* To use more than one named example, you must use the `examples` facet when you include the corresponding NamedExample fragments.
+The reason for using `.yaml` files for single examples and not `.raml` files is that, according to the RAML 1.0 specification, a NamedExample fragment that defines one example is not valid. Therefore, you should use a NamedExample fragment only to define two or more examples.
 
-* All example fragments must have the header "#%RAML 1.0 NamedExample"
-
-Use the `example` facet only if you want to provide a single, unnamed example.
-
-
-== Naming a single example for a type by using an `examples` facet
-
-In the following sample files, the `examples` facet names the example `fullName`.
+For example, the following NamedExample fragments are invalid:
 
-.api.raml
+.fragment1.raml
 ----
-#%RAML 1.0
-title: test
-types:
-  A:
-    properties:
-      givenName: string
-      familyName: string
-    examples:
-      fullName: !include fragment.raml
+#%RAML 1.0 NamedExample
+givenName: ”Chiaki”
+familyName: "Mukai"
 ----
 
-.fragment.raml
+.fragment2.raml
 ----
 #%RAML 1.0 NamedExample
-givenName: ”Chiaki”
-familyName: "Mukai"
+givenName: "Kyung-won"
+familyName: "Park"
 ----
 
+API Designer ignores the header and uses the examples. However, if you validate a specification with any other RAML processors, NamedExample fragments that define only one example should generate errors, in accord with the RAML 1.0 specification.
```

Comment:
```suggestion
API Designer ignores the header and uses the examples. However, if you validate a specification by using any other RAML processors, NamedExample fragments that define only one example generate errors, in accord with the RAML 1.0 specification.
```

Reply:
Do we need "in accord..."? Does the audience need to know this?

Reply:
I have to use "should" here because we cannot definitively say what non-MuleSoft RAML processors do.

And yeah, we do need to state the "in accord" part. We're indirectly saying that API Designer is not fully compliant with RAML 1.0. We have to say this, but we don't want to put a sharp edge on it.